### ISSUE TITLE
Alt text for featured Image in templates

### ISSUE THREAD

**Issue by victorvalle** [liferay-design/liferay.design#597]
Right now we don't have an alternative text to describe featured images in templates. This is an accessibility issue we could easily solve.
The text shown right now is "image for $Title"

 
![image](https://user-images.githubusercontent.com/21121115/68856006-c7eea180-06df-11ea-86af-17e6cc7abf6a.png)

**Comment by plhnk** [comment]
are you proposing an alternative to the default (`image for $title`) or a field where the author inserts alt text? or something else?

**Comment by victorvalle** [comment]
I'm proposing to have and $altFeaturedImage where to put the description if necessary. When not necessary it should be "". So the final result should be "" or $altFeaturedImage

**Comment by plhnk** [comment]
so have a field for the author to insert alt text — if they don't then have nothing ?

**Comment by victorvalle** [comment]
Sometimes there is no need to fill an alt text for the image. In that case, it should be blank.
Take a look at this video in the moment shared:
https://youtu.be/6mhcgHG8lGo?t=576
